how-to/integrate-with-workspace/framework/OpenFin.WPF.TestHarness/MainWindow.xaml.cs

```diff
@@ -18,27 +18,32 @@ public MainWindow()
             InitializeComponent();
             NameValueCollection appSettings = ConfigurationManager.AppSettings;
             string workspaceChannelId = appSettings.Get("workspaceChannelId") ?? Settings.DefaultWorkspaceChannelId;
+            
             // get command line arguments
             string[] args = Environment.GetCommandLineArgs();
-            // check if args are empty
-            if (args.Length > 1) 
+            if (args.Length > 1 && workspaceChannelId == "") 
```

Comment:
PJ could this be one if check?

if(workspaceId == "") {
check command line args and use that if available
else launch popup window
}